### ISSUE TITLE
Preload the last bench of the master branch by default

### ISSUE THREAD

**Issue by snicolet** [glinscott/fishtest#236]
Suggestion: 

When a user submit a test, if the "Base branch" field is "master", then the site store the "Base signature" in a database or in a text file.

Afterwards, on subsequent calls of the http://tests.stockfishchess.org/tests/run page, both the "Base branch" field and the "Base signature" field are pre-loaded with default values:

• Base branch  = "master"
• Base signature = the last signature found in the database.

**Comment by ppigazzini** [comment]
Each PR passes a Travis test, [here an example](https://travis-ci.org/official-stockfish/Stockfish/jobs/365804811), so we could change the Travis script to write the signature value in a convenient place (e.g. a text file, a call to a new fishtest server API).

**Comment by snicolet** [comment]
The scripts of the pull requests would change the bench before it is actually committed to the master branch, which is not what we want because there can be a delay between the pull request and its acceptation. Or the pull request could be rejected, which shouldn't change the bench either.

The solution I proposed is technically simpler, as it is the community as a whole which will change the stored bench when the master branch changes. And my solution will also serve as an implicit incitation for people to synchronize their master branch to the latest master branch, because they will realize that the default bench proposed by the majority doesn't agree with their old master branch.

**Comment by Stefano80** [comment]
The problem I see with that is that one is then not allowed to be even a single commit behind master. What it could be also maybe interesting is to make approval pending on the submitter branch passing Travis integration test. 

This would force all submitters to register to Travis, connect their repo to Travis CI and always put the bench on the commit message. But this is maybe a good thing to do. Any idea?

**Comment by snicolet** [comment]
And I would prefer to try the simplest approach first and test how it feels first for a couple of weeks, before inventing more complicated solutions forcing to maintain any Travis CI code, let alone forcing people to register to Travis and connect their repo to Travis, which would add friction for beginners before they manage to test any patch.

**Comment by snicolet** [comment]
@tomtor That's another possibility, if you think that is simpler and easier to maintain.

**Comment by tomtor** [comment]
@snicolet Yes, currently fishtest already reads some commit info from github when a new run is created, so it is not a big change to read and interpret some more to prefill some form fields.
